<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementarySurfaceTab</class>
 <widget class="QWidget" name="ElementarySurfaceTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Этапы обработки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Номер ЭП</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>831</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="tabKeyNavigation">
    <bool>true</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>44</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row/>
   <column>
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Тип ЭП</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <row>
    <property name="text">
     <string>s1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>s2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>123456</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsEditable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>291</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>52</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>s1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>123456</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметры ЭП</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Количество ЭП</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>601</width>
     <height>111</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>РИ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Тип РИ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Режимы резания</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Стойкость РИ</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Vmin</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vmax</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Smin</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Smax</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tmax</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tmax</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>min</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>max</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>399</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Режимы резания</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
